includes/Modules/Analytics_4.php

```diff
@@ -245,7 +245,97 @@ public function register() {
 		// Analytics 4 tag placement logic.
 		add_action( 'template_redirect', array( $this, 'register_tag' ) );
 
-		$this->get_settings()->on_change(
+		$this->register_property_change_handlers();
```

Comment:
This diff in `register()` is the result of lifting the `on_change()` handler and `pre_update_option_googlesitekit_analytics-4_settings` filter hook registration out to the new `register_property_change_handlers()` method.